modules/dynamic-plug-ins-get-started.adoc

```diff
@@ -0,0 +1,45 @@
+// Module included in the following assemblies:
+//
+// * web_console/dynamic-plug-ins.adoc
+
+:_content-type: PROCEDURE
+[id="getting-started-with-dynamic-plugins_{context}"]
+= Getting started with dynamic plug-ins
+
+In order to get started using the dynamic plug-in, you must set up your environment to write a new OpenShift Console dynamic plug-in.
+
+.Prerequisites
+* Ensure you have link:https://nodejs.org/en/[`Node.js`] and link:https://yarnpkg.com/[`yarn`] installed.
```

Comment:
I think you can break this into two bullets and just use "package_name installed."